home/views.py

```diff
@@ -12,19 +12,33 @@ def landingpage(request):
     return render(request, 'home/landingpage.html')
 
 
-def displayQuestion(request, **kwargs):
+def displayQuestionPage(request, **kwargs):
```

Comment:
Tthis is not a Pythonic function name

Also why not define the args with explicit names instead of parsing to opaque `kwargs` structure?

Reply:
I added some functions to this function, because my feature is in the same view as Lior's feature.
the use of kwargs is was on his PR

Reply:
1. Blaming your friend is not nice.
2. If its broken - fix it.

Reply:
not blaming him, just afraid to touch his code (that already merged) 
but I'll do it

Reply:
But you did touch it... Also why fear, you have tests no? Don't you trust the tests to tell you if you've broken anything important?

Reply:
you right, fixed.